lib/godwoken_explorer/graphql/sourcify.ex

```diff
@@ -91,11 +91,16 @@ defmodule GodwokenExplorer.Graphql.Sourcify do
 
       result =
         if smart_contract do
+          params = filter_params_nil(attrs)
+
           smart_contract
-          |> SmartContract.changeset(attrs)
+          |> SmartContract.changeset(params)
```

Comment:
Would it be better to fitler `nil` out in changeset?

Reply:
changeset function always use for common scenario when update mapping struct(in here is SmartContract).
if we want clear the field of struct that use nil value for changeset params to replace stale value